lib/Api/UserInvitationApi.php

```diff
@@ -17,7 +17,7 @@
  * OpenAPI spec version: v1
  * 
  * Generated by: https://github.com/swagger-api/swagger-codegen.git
- * Swagger Codegen version: 2.4.0-SNAPSHOT
+ * Swagger Codegen version: 2.4.10
  */
```

Comment:
Code changes (mostly just the array/stdClass check change, left another comment where there are some changes for API changes).